vizro-core/docs/pages/user_guides/container.md

```diff
@@ -105,3 +126,173 @@ To add a [`Container`][vizro.models.Container] to your page, do the following:
         [![Container]][Container]
 
     [Container]: ../../assets/user_guides/components/containers.png
+
+
+## Nested containers
+Containers can be nested, providing a hierarchical structure for organizing components.
+This nesting capability allows users to create more complex layouts and manage related components at any level of granularity.
+
+To create nested containers, simply add a `Container` to the `components` argument of another `Container`.
+
+!!! note
+
+    Note that an almost identical layout can also be achieved using [`Page.layout`](layouts.md), see the
+    [advanced grid example](layouts.md#grid-advanced-example) on how this can be done. Here we use `Container` instead
+    of `Page.layout` because we need finer control of selected grid areas (e.g. adding different titles to each subgrid
+    and have different spacing between groups of components).
```

Comment:
```suggestion
  In the below example, we have two groups of charts (left and right) that we would like to arrange separately on the `Page` while also providing a separate title for the two groups. This is often the case when some charts are topically related. In this case it makes sense to use `Containers` instead of `Page.layout`.
  
  Note that an almost identical layout can also be achieved using solely the [`Page.layout`](layouts.md) -  see the [advanced grid example](layouts.md#grid-advanced-example) on how this can be done.
```

Here is an alternative which is more user friendly I think. Before I was not quite sure if I would have understood. Too much talk of nesting, hierarchical, grid etc :P

Reply:
Thanks! 🚀  I've changed the order and rephrased a bit. What I would like the users to read first is that a similar layout can be achieved in a simpler way. And then afterwards explaining why we've decided to go for the Containers in this case. WDYT?

  ```
  Note that an almost identical layout can also be achieved using solely the [`Page.layout`](layouts.md) - 
    see the [advanced grid example](layouts.md#grid-advanced-example) on how this can be done.
    
    Here we use the `Containers` instead of `Page.layout`, as we want to provide clearer visual separation between two chart groups.
    In the below example, we have two groups of charts (left and right) and want to assign separate titles to each group
    and increase the gap between them for enhanced visual clarity. Such an arrangement is particularly advantageous 
    when grouping charts that are thematically related.
```

Reply:
Hm I am not sure I like it. But I think we should also see it in the context of the below comment where after discussing with you I am even more of the opinion that the mention of nesting in combination with this example is actually confusing. So let's not iterate on this until we decide the below.

For this specific case:
- I think the admonition above the example looks visually not great, and it is also too big and long now
- The first sentence refers to something that follows a fair bit below, it does not make sense as it stands - I think actually the opposite: a user should understand first what they are supposed to be wanting to achieve, then see the example, then get notified (no need for big stand-out) that they can also achieve this with an alternative

My overall vote would go to:
- Rename this to "Advanced containers"
- kill the paragraph about nested containers (and move to next section - see last point)
- Use something like your second paragraph or my suggestion to frame the problem/idea
- Note subtly that you can achieve this elsehow
- Have another section about "Nested containers" with a 2-liner and maybe an inline example (ie not a big admonition, just some showcase of a `vm.Container` inside a `vm.Container`

Reply:
P.S. if you are attached to the current example after spending lots of time on it then fine to make a new section "Advanced containers" and keep it in there like @maxschulz-COL suggested 😀 Alternatively we can just shelve it for now and bring it back with controls when we have added them.

Reply:
Thanks for your input - I would just shelve it for now! 👍